tests/helixprep.proj

```diff
@@ -0,0 +1,152 @@
+<Project ToolsVersion="12.0" DefaultTargets="ArchiveAll" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <UsingTask TaskName="ZipFileCreateFromDirectory" AssemblyFile="$(ToolsDir)\net45\Microsoft.DotNet.Build.Tasks.dll"/>
+  
+  <Import Project="dir.props" />
+  <Import Project="..\dir.props" />
+
+  <PropertyGroup>
+    <DiscoveryDirectory>$(TestWorkingDir)</DiscoveryDirectory>
+    <CoreRootDir Condition="'$(RuntimeID)' == '' ">$(CORE_ROOT)</CoreRootDir>
+    <CoreRootDir Condition="'$(RuntimeID)' != '' ">$(CORE_OVERLAY)</CoreRootDir>
+    <CoreRootName Condition="'$(RuntimeID)' == '' ">Core_Root_Windows_NT-$(__BuildArch)</CoreRootName>
+    <CoreRootName Condition="'$(RuntimeID)' != '' ">Core_Root_$(RuntimeID)</CoreRootName>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <TestCmds Include="$(DiscoveryDirectory)\**\*.cmd" ></TestCmds>
+    <XunitDlls Include="$(DiscoveryDirectory)\*\*.XUnitWrapper.dll" ></XunitDlls>
+  </ItemGroup>
+
+  <!-- Build the platform-specific executable to run an individual xunit wrapper -->
+
+  <Target Name="GenerateWrapperExecutables"
+    Inputs="@(XunitDlls)"
+    Outputs="$(TestWorkingDir)*\runtests.cmd" >
+
+    <MSBuild Projects="helixprep.proj"
+             Properties="BuildPath=%(XunitDlls.RootDir)%(XunitDlls.Directory);ProjectName=%(XunitDlls.Filename)%(XunitDlls.Extension)"
+             Targets="GenerateWrapperCmd" />
+
+    <MSBuild Projects="helixprep.proj"
+             Properties="BuildPath=%(XunitDlls.RootDir)%(XunitDlls.Directory);ProjectName=%(XunitDlls.Filename)%(XunitDlls.Extension)"
+             Targets="GenerateWrapperSh" />
+  </Target>
+
+  <!-- Zip each top-level test folder to send to Helix -->
+
+  <Target Name="ArchiveTests"
+    Inputs="@(XunitDlls)"
+    Outputs="$(TestWorkingDir)archive\**" >
+    
+    <Copy SourceFiles="$(CORE_ROOT)\xunit.console.netcore.exe"
```

Comment:
Why copy this with every Xunit DLL?  I'd expect this to be part of the correlation payload, not a workitem one.